src/openrct2/platform/Crash.cpp

```diff
@@ -320,7 +322,11 @@ static std::wstring GetDumpDirectory()
 {
     char userDirectory[MAX_PATH];
     platform_get_user_directory(userDirectory, nullptr, sizeof(userDirectory));
-    auto result = String::ToWideChar(userDirectory);
+
+    auto path = Path::Combine(userDirectory, "crash");
```

Comment:
We have enums for these types of directory.

Called like so: env->GetDirectoryPath(OpenRCT2::DIRBASE::OPENRCT2, OpenRCT2::DIRID::CRASH);